enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult.tsx

```diff
@@ -22,6 +23,46 @@ export interface InteractiveQuestionResultProps {
 
 type QuestionView = "notebook" | "filter" | "summarize" | "visualization";
 
+const VisualizationResultView = () => {
+  const [isVisualizationSelectorOpen, { toggle: toggleVisualizationSelector }] =
+    useDisclosure();
+
+  return (
+    <Stack>
+      <Group m="sm">
+        <Button
+          compact={true}
+          radius="xl"
+          py="sm"
+          px="md"
+          variant="filled"
+          color="brand"
+          onClick={toggleVisualizationSelector}
+        >
+          <Group>
+            <Icon
+              name={isVisualizationSelectorOpen ? "arrow_left" : "arrow_right"}
+            />
+            <Icon name="eye" />
+          </Group>
+        </Button>
+      </Group>
+      <Group noWrap>
+        {isVisualizationSelectorOpen && (
+          <InteractiveQuestion.ChartTypeSelector />
+        )}
+        {/* 
+              A very hacky thing to force the visualization to resize.
+                there's definitely a better way to do this.
+                 */}
+        <InteractiveQuestion.QuestionVisualization
+          key={String(isVisualizationSelectorOpen)}
+        />
+      </Group>
```

Comment:
The width styling for this seems to be broken on Shoppy. This is rendered with `<InteractiveQuestion questionId={questionId} />`

![CleanShot 2567-10-03 at 06 33 53@2x](https://github.com/user-attachments/assets/eae13594-ca52-4c37-a6a7-22b1146faeac)

On DevTools, it looks like the visualization takes on only the width of the chart type selector.

![CleanShot 2567-10-03 at 06 36 04@2x](https://github.com/user-attachments/assets/55101a99-4a72-43f0-afc4-2a597492739a)